ote_sdk/ote_sdk/entities/annotation.py

```diff
@@ -26,7 +31,15 @@ class Annotation(metaclass=abc.ABCMeta):
     def __init__(
         self, shape: ShapeEntity, labels: List[ScoredLabel], id: Optional[ID] = None
     ):
+
         self.__id_ = ID(ObjectId()) if id is None else id
+        check_input_param_type(
+            RequiredParamTypeCheck(shape, "shape", ShapeEntity),
```

Comment:
Is it possible not to list parameters typings using \_\_annotations__ for example?

Reply:
Thanks, added “check_input_parameters_type” decorator currently works with __annotations__ attribute to get expected types of input parameters